### PR TITLE
hyperlink shortened and added new conversation data

### PR DESCRIPTION
**Pull request by ajinkyaghodake123**
- hyperlink shortened  in router
- added new conversation data and respective intents in domain.yml

### CODE REVIEW

File: bot/domain.yml [hunk 1]
```diff
@@ -186,44 +192,66 @@ responses:
...
+     - type: "response"
+       text: "<span>I understand you want to take IGOT Training-Please visit and select your Role to view relevant training: <a target='_blank' href='https://igot.gov.in/igot/explore-course'> DIKSHA IGOT</a><br><hr><strong>**</strong> Press 0 to go to first menu or press 99 to go back or type your query directly</span>"
+       intent: "IGOT"  
+  
+  utter_state_training:
+  - custom:
+     blocks:
+     - type: "response"
+       text: "<span>I understand you want to take a state training. Please visit and select the Topic, Medium, Class and Subject to view relevant training: <a target='_blank' href='https://diksha.gov.in/explore-course'> DIKSHA STATE_TRAINING </a><br><hr><strong>**</strong> Press 0 to go to first menu or press 99 to go back or type your query directly</span>"
+       intent: "IGOT"
```
Comment (sudhir-j-sapkal) [inline bot/domain.yml]: Change intent

File: bot/domain.yml [hunk 2]
```diff
@@ -186,44 +192,66 @@ responses:
...
-       text: "<span>I understand you want to contribute content to DIKSHA <br>Please visit: <a target='_blank' href='https://vdn.diksha.gov.in/'>https://vdn.diksha.gov.in</a><br><hr><strong>**</strong>Press 0 to go to first menu or press 99 to go back or type your query directly</span>"
+       text: "<span>I understand you want to contribute content to DIKSHA <br>Please visit: <a target='_blank' href='https://vdn.diksha.gov.in/'>VIDYADAAN</a><br><hr><strong>**</strong>Press 0 to go to first menu or press 99 to go back or type your query directly</span>"
        intent: "vidyadaan_link"
 
+  utter_IGOT:
+  - custom:
+     blocks:
+     - type: "response"
+       text: "<span>I understand you want to take IGOT Training-Please visit and select your Role to view relevant training: <a target='_blank' href='https://igot.gov.in/igot/explore-course'> DIKSHA IGOT</a><br><hr><strong>**</strong> Press 0 to go to first menu or press 99 to go back or type your query directly</span>"
+       intent: "IGOT"
```
Comment (sudhir-j-sapkal) [inline bot/domain.yml]: make intent small letter

File: bot/domain.yml (hunk 1 above)
Comment (ajinkyaghodake123) [reply bot/domain.yml]: done

File: bot/domain.yml (hunk 2 above)
Comment (ajinkyaghodake123) [reply bot/domain.yml]: done

**Pull request merged**